mmpose/apis/inference_tracking.py

```diff
@@ -218,7 +218,8 @@ def get_track_id(results,
         track_id, results_last, match_result = _track(result, results_last,
                                                       tracking_thr)
         if track_id == -1:
-            if np.count_nonzero(result['keypoints'][:, 1]) > min_keypoints:
+            if min_keypoints <= 0 or np.count_nonzero(
```

Comment:
```suggestion
            if np.count_nonzero(result['keypoints'][:,1]) > min_keypoints:
```
I think we don't need to make ``min_keypoints==0`` a special case.

Reply:
I will try to reproduce it, but I am relatively sure that I received an occasional exception without the explicit handling of min_keypoints (months ago when I first discovered and patched this PR locally on my machine), but I can't 100% recall the situation. I think it may have been if a bounding box ended up with zero keypoint detections (rare case but possible) then result['keypoints'] ends up as None causing the following slice to error. Now that I think about it though, if that is truly the case then it would be better to explicitly handle the None case? I'm not sure, I'll have to see if I can make the exception happen again.

Reply:
I tried to reproduce the exception I described, but wasn't able to do so, so I guess the `min_keypoints == 0` special case is not required.